src/server/common.cc

```diff
@@ -24,11 +25,31 @@ extern "C" {
 #include "server/server_state.h"
 #include "server/transaction.h"
 
+ABSL_FLAG(bool, lock_on_hashtags, false,
+          "When true, locks are done in the {hashtag} level instead of key level. "
+          "Only use this with --cluster_mode=emulated|yes.");
+
 namespace dfly {
 
 using namespace std;
 using namespace util;
 
+bool KeyLockArgs::IsLockHashTagEnabled() {
+  thread_local bool is_enabled_flag_cache = []() {
```

Comment:
Why the lambda here?

Reply:
Leftover, no longer needed. I used the lambda in a previous iteration to execute multiple statements.